pkg/lifecycle/startup/startup.go

```diff
@@ -312,10 +320,32 @@ func MonitorIPAddressSubnets() {
 
 	pollInterval := getMonitorPollInterval()
 
+	var clientset *kubernetes.Clientset
+	var config *rest.Config
+	var k8sNode *v1.Node
+	var err error
+	if config, err = rest.InClusterConfig(); err != nil {
```

Comment:
This code needs to work on non-kubernetes clusters as well - I'd suggest modeling after the logic above: https://github.com/projectcalico/node/pull/1242/files#diff-e481557bd1e2fed4cea80458ebb02e18797fdf49c42944f4a8d6c23cbfbec3f9R130

Reply:
modified as per suggestion